abstractions/php/composer.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
if you have a feed source file, you might want to add this registry to the source (without the username/token) https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#authenticating-with-a-personal-access-token